patches/tModLoader/Terraria/DataStructures/PlayerDeathReason.cs.patch

```diff
@@ -32,6 +32,8 @@
 -		}
 -	}
 -
```

Comment:
Not your patch from this change, but while you're here, change to block comment instead of removal?